pkg/config/config.go

```diff
@@ -91,6 +88,7 @@ func (idConfig *IdentityConfig) loadFromENV() error {
 	loadEnv("TOKEN_REFRESH_INTERVAL", &idConfig.rawTokenRefresh)
 	loadEnv("TOKEN_EXPIRY", &idConfig.rawTokenExpiry)
 	loadEnv("TOKEN_SERVER_ADDR", &idConfig.TokenServerAddr)
+	loadEnv("TOKEN_SERVER_API_ENABLE", &idConfig.rawTokenServerAPIEnable)
```

Comment:
WIP

Reply:
just rename the ENV `TOKEN_SERVER_API_ENABLE`? or, all related fields?

Reply:
Ideally all of them, if you decide to apply.
Totally optional, and up to you.

Reply:
`TOKEN_SERVER_API_ENABLE` => `TOKEN_SERVER_REST_API`? seems simpler.

Reply:
TOKEN_SERVER_REST_API=`true` is definitely easy to understand but
if the key is placed alone like: `TOKEN_SERVER_REST_API`, the user of the application wont be sure which value type to put (is it string boolean) 

In short `TOKEN_SERVER_API_ENABLED` is pretty clear for me that I should put true or false, even without example data.

Still very minor thing though, so its up to you as its your work and we do not have such specific rules.

Reply:
It is difficult to infer the type from the name alone. `TOKEN_SERVER_API_ENABLE` does not tell `true/false` either.
When user setting the option, it is an action so `ENABLE` is natural, but inside the struct, it is a state, so it should be `ENABLED`.🫠 Just to avoid the word to prevent confusion.

flag.BoolVar try-out: https://go.dev/play/p/S5-osKQLF6t

Reply:
renamed in commit: [492b55a](https://github.com/WindzCUHK/k8s-athenz-sia/pull/13/commits/492b55a1b0e2ec53e33d583c9b1fb6e64cc072b3)